apps/dashboard-service/src/app.module.ts

```diff
@@ -0,0 +1,35 @@
+import { Module } from "@nestjs/common";
+import { CommonModule } from "@terramatch-microservices/common";
+import { SentryGlobalFilter, SentryModule } from "@sentry/nestjs/setup";
+import { APP_FILTER } from "@nestjs/core";
+import { HealthModule } from "@terramatch-microservices/common/health/health.module";
+import { TotalSectionHeaderController } from "./dashboard/total-section-header.controller";
+import { TotalSectionHeaderService } from "./dashboard/dto/total-section-header.service";
+import { BullModule } from "@nestjs/bullmq";
+import { CacheService } from "./dashboard/dto/cache.service";
+import { WorkerProcessor } from "./dashboard/worker/worker.processor";
+import { RedisModule } from "@nestjs-modules/ioredis";
+
+@Module({
+  imports: [
+    SentryModule.forRoot(),
+    CommonModule,
+    HealthModule,
+    RedisModule.forRoot({
+      type: "single",
+      url: "redis://127.0.0.1:6379"
+    }),
```

Comment:
If you're going to need direct Redis access, please follow the pattern from the Data API: https://github.com/wri/terramatch-microservices/blob/2fb4a3b02b24bc697784d19af002f80825fc5d27/libs/data-api/src/lib/data-api.module.ts#L9-L19